apps/gpsinfo/gps-info.js

```diff
@@ -100,12 +103,23 @@ function onGPS(fix) {
     layout.maidenhead.label = "Maidenhead: "+maidenhead;
   } else {
     layout.sat.label = fix.satellites;
-    nofix = (nofix+1) % 4;
-    layout.progress.label = ".".repeat(nofix) + " ".repeat(4-nofix);
+    layout.progress.label = "in view: " + SATinView;
   }
   layout.render();
 }
 
+function onGPSraw(nmea) {
+  var nofBD = 0;
+  var nofGP = 0;
+  if (nmea.slice(3,6) == "GSV") {
+    // console.log(nmea);
+    if (nmea.slice(0,7) == "$BDGSV,") nofBD = nmea.slice(11,13);
+    if (nmea.slice(0,7) == "$GPGSV,") nofGP = nmea.slice(11,13);
```

Comment:
```suggestion
    if (nmea.slice(0,7) == "$BDGSV,") nofBD = Number(nmea.slice(11,13));
    if (nmea.slice(0,7) == "$GPGSV,") nofGP = Number(nmea.slice(11,13));
```

When I log SATinView, I sometimes get "0010" because nofBD and/or nofGP are not recognized as numbers.